### PR TITLE
Update claims filing message

### PR DESCRIPTION
**Pull request by cehsu**
This updates the claims filing message.
Please note the following screenshot has been created by hardcoding the changed value into the message variable on the `AdditionalEvidencePage`, like so:
```
      const trackedItem = {displayName: 'Request X'};
      const message = {
        title: 'We have your evidence',
        body: <span>Thank you for sending us {trackedItem ? trackedItem.displayName : 'additional evidence'}. We’ll let you know when we’ve reviewed it.<br/>Note: It may take a few minutes for your uploaded file to show here. If you don’t see your file, please try refreshing the page.</span>
      }

```

screen shot 2017-10-10 at 1 39 47 pm

### CODE REVIEW

File: src/js/claims-status/actions/index.js [hunk 1]
```diff
@@ -221,7 +222,7 @@ export function submitFiles(claimId, trackedItem, files) {
               });
               dispatch(setNotification({
                 title: 'We have your evidence',
-                body: `Thank you for filing ${trackedItem ? trackedItem.displayName : 'additional evidence'}. We’ll let you know when we’ve reviewed it.`
+                body: <span>Thank you for sending us {trackedItem ? trackedItem.displayName : 'additional evidence'}. We’ll let you know when we’ve reviewed it.<br/>Note: It may take a few minutes for your uploaded file to show here. If you don’t see your file, please try refreshing the page.</span>
```
Comment (jbalboni) [inline src/js/claims-status/actions/index.js]: If we're putting jsx in here, we should change the file extension.

**Review by jbalboni — APPROVED** [review]
LGTM with one comment. A screenshot would be helpful, though you might have to use a review instance for that. It's hard to do locally (file uploads don't work in the mock api I gave you).

File: src/js/claims-status/actions/index.js (hunk 1 above)
Comment (cehsu) [reply src/js/claims-status/actions/index.js]: Thanks @jbalboni, changed.

**Comment by cehsu** [comment]
Thanks @jbalboni, I'll include a screen shot as well via review instance as you recommended.

**Pull request merged**